crates/anvil/core/src/eth/mod.rs

```diff
@@ -178,6 +178,12 @@ pub enum EthRequest {
     #[serde(rename = "eth_getTransactionByHash", with = "sequence")]
     EthGetTransactionByHash(TxHash),
 
+    #[serde(rename = "anvil_getBlobByHash", with = "sequence")]
+    GetBlobByHash(TxHash),
```

Comment:
we should use B256 here because this is supposed to be the versioned hash not tx hash?

we can also document this